war/src/main/js/widgets/config/model/ConfigTableMetaData.js

```diff
@@ -50,7 +54,14 @@ function fromConfigTable(configTable) {
     // See the next block of code.
     
     if(!firstRow.hasClass('section-header-row')){
-      var generalRow = $('<tr class="section-header-row insert first" title="General"><td colspan="4"><div class="section-header"><a class="section-anchor">#</a>General</div></td></tr>');
+      var td, tr;
+      if (configTable[0].nodeName === 'TR') {
+        tr = 'tr';
+        td = 'td';
+      } else {
+        tr = td = 'div';
+      }
+      var generalRow = $('<'+tr+' class="section-header-row insert first tr" title="General"><'+td+' colspan="4"><div class="section-header"><a class="section-anchor">#</a>General</div></+'+td+'></'+tr+'>');
```

Comment:
This probably deserves an explanation. I assume this can happen with (some) plugins?

Reply:
the comment is here:
https://github.com/jenkinsci/jenkins/blob/83c1a07f5654ea6de3dbf726178b07ad62dd99c7/war/src/main/js/widgets/config/model/ConfigTableMetaData.js#L52-L54

just out of sight of the diff